### ISSUE TITLE
missing Tailwind 2.0 colors

### ISSUE THREAD

**Issue by CharlesOkwuagwu** [windicss/vite-plugin-windicss#16]
Hi, Thanks for this toolkit.

I have tried this: `const colors = require('windicss/colors')` in tailwind.config.js, but I see these typescript errors.

```
...
colors: {
       ...
        bluegray: colors.blueGray,
        coolgray: colors.coolGray,
        gray: colors.gray,
        truegray: colors.trueGray,
        warmgray: colors.warmGray,
        red: colors.red,
        orange: colors.orange,
}

```

Does `windicss/colors'` provide the full range of Tailwind 2.0 colors?

**Comment by FallDownTheSystem** [comment]
Probably unrelated but extending colors didn't seem to work either.
```js
module.exports = {
	darkMode: false,
	theme: {
		extend: {
			colors: {
				'test-orange': '#e05501'
			}
		}
	}
}
```
I'm not sure if this is intended behavior since the docs do say that "Any configuration options you provide will extend the default options." but the migration guide still shows the theme > extend > colors format.

**Comment by antfu** [comment]
Can anyone share a reproduction with the latest version of this plugin?

**Comment by antfu** [comment]
> > Still no errors were found...Maybe you can publish a repo, let me take a look for you.
> 
> [CharlesOkwuagwu/vite-demo](https://github.com/CharlesOkwuagwu/vite-demo)

Can't see the problem. Pulled your repo and everything seems to works fine. What are the steps to repro your problem?

**Comment by antfu** [comment]
```ts
<span :class="`${selected ? 'font-semibold' : 'font-normal'} block truncate`">
  {{ COUNTRY }}
</span>
```

@asmimo this should work with the latest version. Have you update it?

**Comment by asmimo** [comment]
```vue
<span :class="`${selected ? 'font-semibold' : 'font-normal'} block truncate`">
  {{ COUNTRY }}
</span>
```
@antfu yeah this works fine now.

Some colors don't work like the one I used below. If colors are used with `@apply`, it works even without extending.
```css
.bg-coolGray-900 {
  @apply bg-coolGray-900;
}

.text-lightBlue-500 {
  @apply text-lightBlue-500;
}
``` 
Yeah. this is how I fixed the issue for now.

I tried to extend directly into `windicssOptions` but it doesn't work either. 
Form plugin is not working too.
Here's the reproduced repo: https://github.com/asmimo/vite-windicss.git
And I am using linux(manjaro) btw
